acceptance/acceptance_test.go

```diff
@@ -209,8 +209,11 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 
 	// Start a new server with a custom configuration if the acceptance test
 	// specifies a custom server stubs.
+	var server *testserver.Server
 	if len(config.Server) > 0 {
-		server := testserver.New(t)
+		server = testserver.New(t)
+		server.RecordRequests = true
```

Comment:
Should be configuration on test.toml / Config. I suggest "Record bool".

Reply:
Added `RecordRequests` to the test configuration.